test/test_core_extractor.py

```diff
@@ -173,7 +155,7 @@ def test_cmd_inputs(self):
 
     def test_extractor_cmd(self):
         """Function to test command line working"""
-        cmd = 'python -m functiondefextractor.extractor_cmd --p "%s"' \
+        cmd = 'python -m functiondefextractor --p "%s"' \
```

Comment:
Please make sure not to over wright the files. This change was incorporated in the previous merge !

Reply:
Corrected